src/tribble/transform.py

```diff
@@ -28,14 +29,20 @@
 
 
 def transform(df: pandas.DataFrame) -> pandas.DataFrame:
-    df = _apply_transform_list(df, TRANSFORMERS)
-    return df.groupby('uuid').apply(_group_transform_df)
+    print("Applying " + str(len(TRANSFORMERS)) + " regular transformers:")
+    df = _apply_transform_list(df, TRANSFORMERS, show_progress=True)
+    uuid_grouped = df.groupby('uuid')
+    tqdm.pandas()
+    print("Applying group transformers to " + str(len(uuid_grouped)) + " uuids " + \
```

Comment:
f-strings here too. Also, I don't believe you need the `\` since you're inside parentheses.